dig_core/lib/src/domain/enum/enum.dart

```diff
@@ -1 +1,8 @@
-export 'proposal_status_type.dart';
+enum ProposalStatusType {
```

Comment:
I don't think declaring all enums inside a class is a good idea. From time to time, this file will be very very long. try to avoid that please.

Reply:
I agree.
As you said the above and I have a bit of research.

https://softwareengineering.stackexchange.com/questions/178733/is-it-a-bad-practice-to-include-all-the-enums-in-one-file-and-use-it-in-multiple